docs/dev_guide/index.rst

```diff
@@ -16,26 +16,54 @@ If you are thinking of contributing to sunpy please read the following carefully
 We currently recommend the :ref:`newcomers` as the place to start.
```

Comment:
This preamble could probably do with a tweak given everything else.

Reply:
How about a really short and useless one?